File: Jarvix.Core/ServerError.cs

namespace Ultrapulito.Jarvix.Core {

    public class ServerError {

        #region PROPRIETA'
        public bool IsError { get; set; }
        public string Message { get; set; }
        public string Source { get; set; }
        public string StackTrace { get; set; }
        #endregion

        #region COSTRUTTORI
        public ServerError() {
            this.IsError = true;
            this.Message = "";
            this.Source = "";
            this.StackTrace = "";
        }
        #endregion

    }

}

File: TXIntegrator.View/Javascript/Jarvix/Core/ServerError.js
dojo.provide("Jarvix.Core.ServerError");
dojo.declare("Jarvix.Core.ServerError", null, {

    Data: {
        IsError: null,
        Message: null,
        Source: null,
        StackTrace: null
    },

    Show: function () {
        alert(this.Data.Message);
    }

});


